message_app/model.py

```diff
@@ -2,8 +2,25 @@
 from __future__ import annotations
 from message_app.db.db import DB as db
 import json
+import time
 from typing import Union, List
 
+# TODO: Optimize all models by removing the duplicated codes.
+# All the models will be REFACTORED in the future for more functionalities
+# and better understanding. For now, these models are good enough for
+# our upcoming prototype.
+
+# A join table for the many-to-many relationship between users and conversations tables
+# For now, we don't need to create a model for this table because this table only serves
```

Comment:
And I don't think we ever need to create a model for it. So this can be something like `(create a model for this table is unnecessary)` is enough